app/partials/coinify/summary.pug

```diff
@@ -1,8 +1,8 @@
 .flex-column.mbl(ng-show="error || $parent.error")
   p.mvm.state-danger-text.type-md(translate="UNABLE_TO_PROCESS")
   span.mvm(ng-hide="isSell" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
-  span.mvm(ng-show="isSell && !$parent.error.error === 'price_quote_expired'" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
-  span.mvm(ng-show="isSell && $parent.error.error === 'price_quote_expired'" translate="SELL.QUOTE_EXPIRED_ERROR" translate-values="{error: $parent.error.error_description}")
+  span.mvm(ng-show="isSell && $parent.$parent.error.error !== 'price_quote_expired'" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.$parent.error.error_description}")
```

Comment:
this is gross and will be refactored so that sell's summary page and controller is separated out from buy. but for now, this is needed.